Tests/MapboxMapsTests/Annotations/Generated/PolygonAnnotationManagerTests.swift

```diff
@@ -502,6 +524,12 @@ final class PolygonAnnotationManagerTests: XCTestCase, AnnotationInteractionDele
             ]]))
         ]
 
+        manager.annotations = manager.annotations.map { annotation in
+            var annotation = annotation
+            annotation.isDraggable = true
```

Comment:
This seems to be very unergonomic to use, could you add `isDraggable`/`isSelected` to *Annotation initializer to remedy this? I believe it won't be a breaking change, as long as there is a default value(Swift API Digester may disagree).